data/Eric-Obeng.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Eric Obeng",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "A Frontend Developer passionate about learning new technologies to help solve problems",
+  "avatar": "https://github.com/Eric-Obeng.png",
```

Comment:
Hey!! No need to add avatar here. It will be auto imported from your github profile.